pkg/endpoint/endpoint.go

```diff
@@ -2275,3 +2277,16 @@ func (e *Endpoint) MapPinLocked() error {
 
 	return err
 }
+
+// InitSysctl configures sysctl settings needed for IPSec
+func (e *Endpoint) InitSysctl() error {
+	if option.Config.EnableIPSec == false {
+		return nil
+	}
+
+	ip4ConfPath := "/proc/sys/net/ipv4/conf/"
```

Comment:
This is already being done here https://github.com/cilium/cilium/blob/7bfe015f30006a18e64f288518cbcba432e754f0/pkg/endpoint/connector/veth.go#L85

Reply:
my tests were not disabling rp_filter so... I guess need to sort out where this is enabled/disabled.

Reply:
OK not needed on master I'll track back through 'git blame' and see why my original scripts needed it. Perhaps it was just being overly paranoid at the time or something about my setup. Anyways popped the patch off the stack here. Thanks, nice catch.